core/corehttp/commands.go

```diff
@@ -51,9 +51,6 @@ func addCORSFromEnv(c *cmdsHttp.ServerConfig) {
 	origin := os.Getenv(originEnvKey)
 	if origin != "" {
 		log.Warning(originEnvKeyDeprecate)
-		if len(c.AllowedOrigins()) == 0 {
-			c.SetAllowedOrigins([]string{origin}...)
-		}
```

Comment:
@whyrusleeping just double checking... this wasn't intentional, was it?

Reply:
No, something about that seems wrong.